keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -201,11 +201,13 @@ Currently, null values trigger an error when nullable is false and don't
 get defaulted if a default is available. We propose to start defaulting
```

Comment:
can't comment above, but I would clarify the first paragraph of the proposal like this:

```diff
It is important to us that the OpenAPI definition of a type should
result in the same behavior, whether the type is built-in or a CRD.

In order to do that, we need to change how CRDs are defaulted, and
+ place specific requirements on definition of built-in types
+ (specifically around omitempty on non-pointer scalars,
+ and an implicit default of {} on non-pointer struct fields), and
create a defaulting mechanism for built-in that would behave the same
```